EvalServer/src/utils/run_evaluation.py

```
@@ -800,55 +1001,105 @@ def __init__(self):
-            # G-Eval variants
-            "g_eval_correctness": {"display": "Answer Correctness", "camel": "answerCorrectness"},
-            "g_eval_coherence": {"display": "Coherence", "camel": "coherence"},
-            "g_eval_tonality": {"display": "Tonality", "camel": "tonality"},
-            "g_eval_safety": {"display": "Safety", "camel": "safety"},
+            "action_relevance": {"display": "Action Relevance", "camel": "actionRelevance"},
+            "planning_quality": {"display": "Planning Quality", "camel": "planningQuality"},
+        }
+        
+        # Conversational metric names (for multi-turn datasets)
+        # These map to the same config keys but have different display names
+        conversational_metric_map = {
```

Ruff (0.14.8)

Local variable `conversational_metric_map` is assigned to but never used

Remove assignment to unused variable `conversational_metric_map`

(F841)